src/Pode.psm1

```diff
@@ -31,4 +31,4 @@ Get-ChildItem "$($root)/Public/*.ps1" | ForEach-Object { . ([System.IO.Path]::Ge
 $funcs = Get-ChildItem Function: | Where-Object { $sysfuncs -notcontains $_ }
 
 # export the module's public functions
-Export-ModuleMember -Function ($funcs.Name)
\ No newline at end of file
+Export-ModuleMember -Function ($funcs.Name) -Alias Enable-PodeOA,Enable-PodeOpenApiViewer
```

Comment:
It's possible to make this work in a simple fashion to `$funcs.Name` but for aliases:

Where `$sysfuncs = Get-ChildItem Function:` is set, add the following line to fetch all aliases:
```powershell
$sysaliases = Get-ChildItem Alias:
```

then after the `$funcs = Get-ChildItem Function: | Where-Object { $sysfuncs -notcontains $_ }` line, the following line to filter down to just Pode aliases:
```powershell
$aliases = Get-ChildItem Alias: | Where-Object { $sysaliases -notcontains $_ }
```

which means the `Export-ModuleMember` can be the following 😄 
```powershell
Export-ModuleMember -Function $funcs.Name -Alias $aliases.Name
```

Reply:
The `$sysaliases = Get-ChildItem Alias:` is missing :)

Reply:
oppss